internal/xds/rbac/rbac_engine_test.go

```diff
@@ -424,16 +432,200 @@ func (s) TestNewChainEngine(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "SimpleAuditLogger",
+			policies: []*v3rbacpb.RBAC{
+				{
+					Action: v3rbacpb.RBAC_ALLOW,
+					Policies: map[string]*v3rbacpb.Policy{
+						"anyone": {
+							Permissions: []*v3rbacpb.Permission{
+								{Rule: &v3rbacpb.Permission_Any{Any: true}},
+							},
+							Principals: []*v3rbacpb.Principal{
+								{Identifier: &v3rbacpb.Principal_Any{Any: true}},
+							},
+						},
+					},
+					AuditLoggingOptions: &v3rbacpb.RBAC_AuditLoggingOptions{
+						AuditCondition: v3rbacpb.RBAC_AuditLoggingOptions_ON_ALLOW,
+						LoggerConfigs: []*v3rbacpb.RBAC_AuditLoggingOptions_AuditLoggerConfig{
+							{AuditLogger: &v3corepb.TypedExtensionConfig{
+								Name:        "TestAuditLoggerBuffer",
```

Comment:
I should have realized this in the previous PR. Sorry about that. So when you convert the config in authz policy into protobuf.Any (https://github.com/grpc/grpc-go/blob/89bc83349db7d89870f406b49768777a78fd9dbd/authz/rbac_translator.go#L309), you need to construct a [TypedStruct](https://www.envoyproxy.io/docs/envoy/latest/xds/type/v3/typed_struct.proto). And the typed_url for that TypedStruct needs to be something like `grpc.audit_logging/{config.Name}` (the prefix is up to us). When the rbac engine reads audit_logger.typed_config, it will try built-in loggers, which we don't have yet, and then fall back to custom loggers by treating the value of the Any as a TypedStruct. That's how we get the logger name, not this one.

Reply:
This should be addressed by the changes since your last review, PTAL